pkg/metrics/statsd/metrics.go

```diff
@@ -55,7 +55,9 @@ func (f *metric) MarshalStatsd() ([]byte, error) {
 	}
 
 	if f.ValueFloat != nil {
-		_, err = fmt.Fprintf(buf, ":%f|f\n", *f.ValueFloat)
+		// we emit float values as int for better readability.
+		// This drops latency metrics values after comma
+		_, err = fmt.Fprintf(buf, ":%d|f\n", int(*f.ValueFloat))
```

Comment:
shouldn't do this - it's an indication that Gauge should be being used instead.

Reply:
How would you see "Emit gauge as int" implemented? convert on each metric sending? so we always have `123.00000` ?